src/pages/Register.tsx

```diff
@@ -1,42 +1,51 @@
 import { useState, FormEvent } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import { UserPlus } from 'lucide-react';
 import Button from '../components/Button';
 import Input from '../components/Input';
 import toast from 'react-hot-toast';
-import axios from 'axios'
+import { authService } from '../services/api';
 
 export default function Register() {
+  const navigate = useNavigate();
   const [isLoading, setIsLoading] = useState(false);
-  const [firstName, setfirstName] = useState("")
-  
+  const [formData, setFormData] = useState({
+    firstName: '',
```

Comment:
Update the Register page, to take values from the user to post the Register api request respectively, to store the data in the mongo DB server